CORE_AND_ELECTIVE_COURSES/CORE_COURSE_B/README.md

```diff
@@ -24,11 +24,23 @@ In this course, attendees will build a libp2p application. The course will cover
 
 The course will give you the opportunity to build your application for one of three platforms/languages: Go, Node.js, and Web Browsers. See the preparation section below for your respective choice.
 
+1. Have your favorite code editor ready, such as [VSCode](https://code.visualstudio.com)
+
 ### Preparing for Go
-Everyone will need a working go 1.12 installation. Beyond that, feel free to study up on the basic concepts on the [libp2p website](http://docs.libp2p.io/) and study the interfaces in [go-libp2p-core](https://github.com/libp2p/go-libp2p-core). 
 
-### Preparing for Node.js
-`TBW`
+Everyone will need a working go 1.12 installation. Beyond that, feel free to study up on the basic concepts on the [libp2p website](http://docs.libp2p.io/) and study the interfaces in [go-libp2p-core](https://github.com/libp2p/go-libp2p-core).
+
+### Preparing for JS
+
+1. Node.js >= 10 installed
+1. npm >= 6 installed
+1. Download or clone the code at https://github.com/libp2p/js-libp2p-examples, `git clone https://github.com/libp2p/js-libp2p-examples.git`.
+
+#### Node.js
+
+1. Install dependencies by following the Setup directions at https://github.com/libp2p/js-libp2p-examples/tree/master/chat/nodejs#setup.
+
+#### Web Browsers
 
-### Preparing for Web Browsers
-`TBW`
+1. Have a Web Browser installed. The chat example was built and tested with [Firefox](https://www.mozilla.org/firefox/new/).
```

Comment:
Any Browser? Better be explicit and recommend the Browsers and versions that you know work.

Reply:
Firefox is specified